cmflib/metadata_helper.py

```diff
@@ -305,6 +305,7 @@ def create_new_execution_in_existing_context(
 EXECUTION_CONTEXT_NAME_PROPERTY_NAME = "Context_Type"
 EXECUTION_CONTEXT_ID = "Context_ID"
 EXECUTION_EXECUTION = "Execution"
+EXECUTION_EXECUTION_TYPE_NAME="Execution_type_name"
```

Comment:
where are we setting the value ?

Reply:
We are passing this execution_type_name in cmf.py, create_execution and merge_created_execution.

Reply:
![image](https://github.com/HewlettPackard/cmf/assets/111754147/04ec23e8-827c-4292-b8c5-4542ffefe744)